src/main/scala/translating/SemanticsLoader.scala

```diff
@@ -558,6 +569,8 @@ class SemanticsLoader(parserMap: immutable.Map[String, Array[Array[StmtContext]]
       case "__BranchTaken" => None
       case "BTypeNext" => None
       case "BTypeCompatible" => None
+      case "TPIDR_EL0" => Some(Register(name, 64))
```

Comment:
What opcodes produce this?

Reply:
mrs https://katrinafyi.github.io/aslp-web/5/?opcode=0xd53bd040&bytes=40+D0+3B+D5&asm=mrs+x0%2C+TPIDR_EL0&debug=0